### DISCUSSION TITLE
sharing songs currently playing ingame and menu for discord

### DISCUSSION THREAD

**Discussion by UnboundMirror** [ppy/osu#25617] [Ideas] [answered]
i just think it would be nice if you could create a way to send songs ur playing or listening to ingame to discord so they can see what ur listening to similar to spotify

**Comment by bdach** [comment, marked as answer]
This will not be happening. osu! is not a music player.

**Comment by UnboundMirror** [comment]
i know it was just a idea  sorry if u  thought i was being serious
________________________________

**Comment by Joehuu** [comment]
> a way to send songs ur playing

Kinda a thing already with the "view beatmap" button on rich presence (but only when playing).

<img width="299" alt="Screenshot 2023-12-01 at 9 10 17 AM" src="https://github.com/ppy/osu/assets/35318437/a945d9e0-ae7d-4926-b637-2117576f39d8">
